src/screens/ContactScreen/index.js

```diff
@@ -199,9 +199,9 @@ const ContactScreen = ({navigation}) => {
   };
 
   const handleAddParticipant = () => {
-    const newSelectedUsers = selectedUsers;
-    setSelectedUsers([]);
-    onAddMember(newSelectedUsers);
+    if (!isLoadingAddMember) {
+      onAddMember(selectedUsers);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for preventing multiple clicks on the add participant button while it's still processing. However, it would be better to also provide some user feedback when they click the button while `isLoadingAddMember` is true. This could be a simple toast message informing them that their request is being processed.

```diff
  const handleAddParticipant = () => {
    if (!isLoadingAddMember) {
      onAddMember(selectedUsers);
+   } else {
+     // Display a toast message or some other form of user feedback
+     console.log("Adding member in progress. Please wait...");
    }
  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->